tests/integration/mkdocs.yml

```diff
@@ -1,4 +1,4 @@
-strict: True
+strict: False
```

Comment:
Will want to remove this before final merge.

Reply:
Of course, but I'm wondering you got it work within your example.
I normally get this warning:

```
WARNING  -  Documentation file 'index.md' contains a link to '..\..\..\README.md' which is not found in the documentation files.
```

Reply:
> Of course, but I'm wondering you how got it work within your example. I normally get this warning:
> 
> ```
> WARNING  -  Documentation file 'index.md' contains a link to '..\..\..\README.md' which is not found in the documentation files.
> ```

@johnthagen Ok, got it. `mkdocs` complains about missing files but not folders. So, your particular example from #58 works in `strict` mode.